.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Build and run tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 12 ]
+
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Cache node modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-v${{ matrix.node }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}-${{ hashFiles('.github/workflows/test.yml') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-v${{ matrix.node }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}-${{ hashFiles('.github/workflows/test.yml') }}
+          ${{ runner.OS }}-node-v${{ matrix.node }}-
+
+    - name: Install, build and run tests
+      run: sh ./build.sh
```

Comment:
I would prefer to list the three actions explicitly, but I guess no one else will agree with me :(

Reply:
OK, there you have it.